<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="enabled">
   <bool>true</bool>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>550</width>
    <height>400</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>550</width>
    <height>400</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>550</width>
    <height>400</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>X-Courier</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QWidget" name="horizontalLayoutWidget">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>20</y>
      <width>261</width>
      <height>341</height>
     </rect>
    </property>
    <layout class="QHBoxLayout" name="leftLayout">
     <item>
      <widget class="QGroupBox" name="clientBox">
       <property name="title">
        <string>Client details</string>
       </property>
       <widget class="QLabel" name="client_fname">
        <property name="geometry">
         <rect>
          <x>20</x>
          <y>134</y>
          <width>71</width>
          <height>21</height>
         </rect>
        </property>
        <property name="text">
         <string>First name</string>
        </property>
        <property name="buddy">
         <cstring>client_fname_line</cstring>
        </property>
       </widget>
       <widget class="QLabel" name="client_lname">
        <property name="geometry">
         <rect>
          <x>20</x>
          <y>164</y>
          <width>71</width>
          <height>21</height>
         </rect>
        </property>
        <property name="text">
         <string>Last name</string>
        </property>
        <property name="buddy">
         <cstring>client_lname_line</cstring>
        </property>
       </widget>
       <widget class="QLabel" name="client_pkg_destination">
        <property name="geometry">
         <rect>
          <x>20</x>
          <y>254</y>
          <width>121</width>
          <height>21</height>
         </rect>
        </property>
        <property name="text">
         <string>Pac&amp;kage destination</string>
        </property>
        <property name="buddy">
         <cstring>client_destination_combo</cstring>
        </property>
       </widget>
       <widget class="QLabel" name="client_email">
        <property name="geometry">
         <rect>
          <x>20</x>
          <y>194</y>
          <width>59</width>
          <height>21</height>
         </rect>
        </property>
        <property name="text">
         <string>Email</string>
        </property>
        <property name="buddy">
         <cstring>client_email_line</cstring>
        </property>
       </widget>
       <widget class="QLabel" name="client_phone">
        <property name="geometry">
         <rect>
          <x>20</x>
          <y>224</y>
          <width>91</width>
          <height>21</height>
         </rect>
        </property>
        <property name="text">
         <string>Phone number</string>
        </property>
        <property name="buddy">
         <cstring>client_phone_line</cstring>
        </property>
       </widget>
       <widget class="QLineEdit" name="client_fname_line">
        <property name="geometry">
         <rect>
          <x>100</x>
          <y>134</y>
          <width>151</width>
          <height>22</height>
         </rect>
        </property>
       </widget>
       <widget class="QLineEdit" name="client_lname_line">
        <property name="geometry">
         <rect>
          <x>100</x>
          <y>164</y>
          <width>151</width>
          <height>22</height>
         </rect>
        </property>
       </widget>
       <widget class="QLineEdit" name="client_email_line">
        <property name="geometry">
         <rect>
          <x>100</x>
          <y>194</y>
          <width>151</width>
          <height>22</height>
         </rect>
        </property>
       </widget>
       <widget class="QLineEdit" name="client_phone_line">
        <property name="geometry">
         <rect>
          <x>120</x>
          <y>224</y>
          <width>131</width>
          <height>22</height>
         </rect>
        </property>
       </widget>
       <widget class="QComboBox" name="client_destination_combo">
        <property name="geometry">
         <rect>
          <x>150</x>
          <y>254</y>
          <width>101</width>
          <height>22</height>
         </rect>
        </property>
       </widget>
       <widget class="Line" name="line">
        <property name="geometry">
         <rect>
          <x>20</x>
          <y>114</y>
          <width>231</width>
          <height>16</height>
         </rect>
        </property>
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
       </widget>
       <widget class="Line" name="line_2">
        <property name="geometry">
         <rect>
          <x>20</x>
          <y>280</y>
          <width>231</width>
          <height>16</height>
         </rect>
        </property>
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
       </widget>
       <widget class="QLabel" name="label">
        <property name="enabled">
         <bool>true</bool>
        </property>
        <property name="geometry">
         <rect>
          <x>20</x>
          <y>20</y>
          <width>231</width>
          <height>91</height>
         </rect>
        </property>
        <property name="text">
         <string/>
        </property>
        <property name="pixmap">
         <pixmap>dogeicon.png</pixmap>
        </property>
        <property name="scaledContents">
         <bool>false</bool>
        </property>
        <property name="wordWrap">
         <bool>false</bool>
        </property>
        <property name="margin">
         <number>-11</number>
        </property>
        <property name="openExternalLinks">
         <bool>false</bool>
        </property>
        <property name="textInteractionFlags">
         <set>Qt::NoTextInteraction</set>
        </property>
       </widget>
       <widget class="QLabel" name="status_2">
        <property name="geometry">
         <rect>
          <x>20</x>
          <y>293</y>
          <width>221</width>
          <height>31</height>
         </rect>
        </property>
        <property name="text">
         <string/>
        </property>
       </widget>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QWidget" name="horizontalLayoutWidget_2">
    <property name="geometry">
     <rect>
      <x>280</x>
      <y>20</y>
      <width>261</width>
      <height>151</height>
     </rect>
    </property>
    <layout class="QHBoxLayout" name="rightLayout">
     <item>
      <widget class="QGroupBox" name="recipientBox">
       <property name="title">
        <string>Recipient details</string>
       </property>
       <widget class="QLineEdit" name="recipient_email_line">
        <property name="geometry">
         <rect>
          <x>100</x>
          <y>90</y>
          <width>151</width>
          <height>22</height>
         </rect>
        </property>
       </widget>
       <widget class="QLineEdit" name="recipient_fname_line">
        <property name="geometry">
         <rect>
          <x>100</x>
          <y>30</y>
          <width>151</width>
          <height>22</height>
         </rect>
        </property>
       </widget>
       <widget class="QLineEdit" name="recipient_lname_line">
        <property name="geometry">
         <rect>
          <x>100</x>
          <y>60</y>
          <width>151</width>
          <height>22</height>
         </rect>
        </property>
       </widget>
       <widget class="QLabel" name="recipient_fname">
        <property name="geometry">
         <rect>
          <x>20</x>
          <y>30</y>
          <width>71</width>
          <height>21</height>
         </rect>
        </property>
        <property name="text">
         <string>First name</string>
        </property>
        <property name="buddy">
         <cstring>recipient_fname_line</cstring>
        </property>
       </widget>
       <widget class="QLabel" name="recipient_phone">
        <property name="geometry">
         <rect>
          <x>20</x>
          <y>120</y>
          <width>91</width>
          <height>21</height>
         </rect>
        </property>
        <property name="text">
         <string>Phone number</string>
        </property>
        <property name="buddy">
         <cstring>recipient_phone_line</cstring>
        </property>
       </widget>
       <widget class="QLabel" name="recipient_email">
        <property name="geometry">
         <rect>
          <x>20</x>
          <y>90</y>
          <width>59</width>
          <height>21</height>
         </rect>
        </property>
        <property name="text">
         <string>Email</string>
        </property>
        <property name="buddy">
         <cstring>recipient_email_line</cstring>
        </property>
       </widget>
       <widget class="QLabel" name="recipient_lname">
        <property name="geometry">
         <rect>
          <x>20</x>
          <y>60</y>
          <width>71</width>
          <height>21</height>
         </rect>
        </property>
        <property name="text">
         <string>Last name</string>
        </property>
        <property name="buddy">
         <cstring>recipient_lname_line</cstring>
        </property>
       </widget>
       <widget class="QLineEdit" name="recipient_phone_line">
        <property name="geometry">
         <rect>
          <x>120</x>
          <y>120</y>
          <width>131</width>
          <height>22</height>
         </rect>
        </property>
       </widget>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QWidget" name="verticalLayoutWidget">
    <property name="geometry">
     <rect>
      <x>280</x>
      <y>180</y>
      <width>261</width>
      <height>181</height>
     </rect>
    </property>
    <layout class="QVBoxLayout" name="verticalLayout">
     <item>
      <widget class="QGroupBox" name="operatorBox">
       <property name="title">
        <string>Package details</string>
       </property>
       <widget class="QLabel" name="packageWeightLabel">
        <property name="geometry">
         <rect>
          <x>10</x>
          <y>90</y>
          <width>101</width>
          <height>21</height>
         </rect>
        </property>
        <property name="text">
         <string>Package weight</string>
        </property>
        <property name="buddy">
         <cstring>package_weight_line</cstring>
        </property>
       </widget>
       <widget class="QLabel" name="packageTypeLabel">
        <property name="geometry">
         <rect>
          <x>10</x>
          <y>120</y>
          <width>91</width>
          <height>21</height>
         </rect>
        </property>
        <property name="text">
         <string>Package type</string>
        </property>
        <property name="buddy">
         <cstring>typeComboBox</cstring>
        </property>
       </widget>
       <widget class="QLineEdit" name="package_weight_line">
        <property name="geometry">
         <rect>
          <x>120</x>
          <y>90</y>
          <width>131</width>
          <height>21</height>
         </rect>
        </property>
       </widget>
       <widget class="QComboBox" name="typeComboBox">
        <property name="geometry">
         <rect>
          <x>120</x>
          <y>120</y>
          <width>131</width>
          <height>22</height>
         </rect>
        </property>
       </widget>
       <widget class="QLabel" name="transportTypeLabel">
        <property name="geometry">
         <rect>
          <x>10</x>
          <y>150</y>
          <width>91</width>
          <height>21</height>
         </rect>
        </property>
        <property name="text">
         <string>Transport type</string>
        </property>
        <property name="buddy">
         <cstring>transportComboBox</cstring>
        </property>
       </widget>
       <widget class="QComboBox" name="transportComboBox">
        <property name="geometry">
         <rect>
          <x>120</x>
          <y>150</y>
          <width>131</width>
          <height>22</height>
         </rect>
        </property>
       </widget>
       <widget class="QLabel" name="packageWeightLabel_2">
        <property name="geometry">
         <rect>
          <x>10</x>
          <y>30</y>
          <width>101</width>
          <height>21</height>
         </rect>
        </property>
        <property name="text">
         <string>Package name</string>
        </property>
        <property name="buddy">
         <cstring>package_name_line</cstring>
        </property>
       </widget>
       <widget class="QLineEdit" name="package_name_line">
        <property name="geometry">
         <rect>
          <x>120</x>
          <y>30</y>
          <width>131</width>
          <height>21</height>
         </rect>
        </property>
       </widget>
       <widget class="QLabel" name="packageValueLabel">
        <property name="geometry">
         <rect>
          <x>10</x>
          <y>60</y>
          <width>101</width>
          <height>21</height>
         </rect>
        </property>
        <property name="text">
         <string>Package value</string>
        </property>
        <property name="buddy">
         <cstring>package_value_line</cstring>
        </property>
       </widget>
       <widget class="QLineEdit" name="package_value_line">
        <property name="geometry">
         <rect>
          <x>120</x>
          <y>60</y>
          <width>131</width>
          <height>21</height>
         </rect>
        </property>
       </widget>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QWidget" name="horizontalLayoutWidget_3">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>370</y>
      <width>531</width>
      <height>21</height>
     </rect>
    </property>
    <layout class="QHBoxLayout" name="bottomLayout">
     <item>
      <widget class="QPushButton" name="quit">
       <property name="text">
        <string>Quit</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="status">
       <property name="text">
        <string>Check Status</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="send">
       <property name="text">
        <string>Send</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <tabstops>
  <tabstop>client_fname_line</tabstop>
  <tabstop>client_lname_line</tabstop>
  <tabstop>client_email_line</tabstop>
  <tabstop>client_phone_line</tabstop>
  <tabstop>client_destination_combo</tabstop>
  <tabstop>recipient_fname_line</tabstop>
  <tabstop>recipient_lname_line</tabstop>
  <tabstop>recipient_email_line</tabstop>
  <tabstop>recipient_phone_line</tabstop>
  <tabstop>package_name_line</tabstop>
  <tabstop>package_value_line</tabstop>
  <tabstop>package_weight_line</tabstop>
  <tabstop>typeComboBox</tabstop>
  <tabstop>transportComboBox</tabstop>
  <tabstop>send</tabstop>
  <tabstop>status</tabstop>
  <tabstop>quit</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
